packages/shared/user-data/LocalStorage/config.ts

```diff
@@ -10,4 +10,5 @@ export const config = {
   reactDevToolsStateExpanded: Boolean(true),
   replayNextCurrentPanel: "sources" as ReplayNextCurrentPanel,
   replayVideoPanelCollapsed: false,
+  enableTestView: false,
```

Comment:
nit: This could use a more descriptive name. Maybe something like `enableTestSuitesTestsView` or something?